src/api/helpers/scrape-img-qr.ts

```diff
@@ -23,9 +23,8 @@ export async function scrapeImg(page: Page): Promise<ScrapQrcode | undefined> {
     .evaluate(() => {
       const selectorImg = document.querySelector('canvas');
       const selectorUrl = selectorImg.closest('[data-ref]');
-      const buttonReload = selectorUrl.querySelector(
-        '[role="button"]'
-      ) as HTMLButtonElement;
+      //const buttonReload = selectorUrl.querySelector('[role="button"]') as HTMLButtonElement;
+      const buttonReload = selectorUrl.querySelector('[data-testid="refresh-large"]');
```

Comment:
I think this is simpler:
```suggestion
      const buttonReload = selectorUrl.querySelector('button');
```